folly/experimental/hazptr/example/SWMRList.h

```diff
@@ -78,42 +78,45 @@ class SWMRListSet {
     }
   }
 
-  bool add(const T v) {
+  bool add(T v) {
```

Comment:
Since this is an example of hazard pointers, not an example of optimal API design for avoiding accidental copies, this is fine.